gradle.properties

```diff
@@ -18,4 +18,8 @@ jacocoVersion=0.8.6
 wso2OrbitAntlrVersion=4.5.1.wso2v1
 ballerinaGradlePluginVersion=0.14.1
 wso2CommonsPoolVersion=1.5.6.wso2v1
-ballerinaTomlParserVersion=1.2.2
+stdlibIoVersion=1.3.0-20220823-212700-b66f477
+stdlibTimeVersion=2.2.2
+stdlibPersistVersion=0.1.0-20220825-160000-f9550ac
+stdlibSqlVersion=1.5.0-20220824-204200-5a27298
+stdlibMysqlVersion=1.4.1
```

Comment:
```suggestion
stdlibMysqlVersion=1.4.1

```

Reply:
Updated in [895edde](https://github.com/ballerina-platform/persist-tools/pull/15/commits/895eddeb28babc700ff8c472fe1745b73ebe9286)